### ISSUE TITLE
ARC WARDEN SKILL BETWEEN ARC AND HIS TEMPEST IS SWITCHED!!! IT'S HAPPEN SINCE LONG TIME AGO AND NOT YET FIXED!!!

### ISSUE THREAD

**Issue by rizqiro** [ValveSoftware/Dota2-Gameplay#13106]
### Ability name

All Skill

### Description

ALL SKILL OF ARC WARDEN IS SWITCHED BETWEEN ARC AND HIS TEMPEST DOUBLE, MAKING HIS SKILL 2 GIVING HIM ATTACK RANGE AND MAGIC DAMAGE AND HIS TEMPEST'S SKILL 2 GIVING HIM ATTACK SPEED AND EVASION. REALLY ANNOYING SINCE THIS HERO IS MY MAIN!!!

### Example Match ID (and possibly Timestamp)

7402885196 ; 7401806910; 7398746811 (this one is correct)

**Comment by irismus** [comment]
Playing Arc Warden on the Dire side will give you the Tempest's abilities on the Self by default (since 7.33).

**Comment by rizqiro** [comment]
nope, you're totally wrong.. see my replay if you want to see the fact,
even if playing on the dire side would solve the problem, IT WILL STILL A BUG TO SOLVE

**Comment by Aluerie** [comment]
read 7.33.
> The upheaval has attuned Arc Warden to the Ancients. When playing on Dire side, Arc Warden will have the Tempest abilities by default and his Tempest Double will have its normal abilities. This condition is temporary and will go away once Zet adapts to the new paradigm.
